src/app/infrastructure/rpc-edge/get-started/page.mdx

```diff
@@ -0,0 +1,169 @@
+  ```jsx
+  import { ThirdwebProvider } from "@thirdweb-dev/react";
+
+function MyApp() {
+  return (
+    <ThirdwebProvider
+      activeChain="ethereum"
+      clientId="your-client-id"
+    >
+      <YourApp />
+    </ThirdwebProvider>
+  );
+}
+```
+</TabsContent>
+  <TabsContent value="typescript">
+  ```ts
+    // Read-only mode
+    const readOnlySdk = new ThirdwebSDK("<chain_id>", {
+      secretKey: "<your_secret_key>",
+    });
+  ```
+
+    When using the Typescript SDK for frontend applications, use the client id:
+
+  ```ts
+    const readOnlySdk = new ThirdwebSDK("<chain_id>", {
+    clientId: "<your_client_id>",
+    });
+  ```
+  </TabsContent>
+  <TabsContent value="python">
+  When using the Python SDK, use the secret key to instantiate the SDK:
+
+  ```python
+  sdk = ThirdwebSDK.from_private_key(
+      <private_key>,
+      "<chain_id>",
+      SDKOptions(secret_key=<your_secret_key>)
+  )
+  ```
+  </TabsContent>
+  <TabsContent value="go">
+  When using the GO SDK, you can use the secret key to instantiate the SDK:
+
+  ```go
+    func main() {
+        // Your secret key from the thirdweb api keys dashboard
+        secretKey := "<your_secret_key"
+        sdk, err := thirdweb.NewThirdwebSDK("<chain_id>", &thirdweb.SDKOptions{
+          SecretKey: <your_secret_key>,
+        })
+      }
+  ```
+</TabsContent>
+  <TabsContent value="unity">
+  When using the Unity SDK, you can use the client id to instantiate the SDK:
+
+  ```csharp
+    ThirdwebSDK sdk = new ThirdwebSDK("<chain_id>", 5, new ThirdwebSDK.Options()
+    {
+      clientId = "<your_client_id>",
+    });
+  ```
+  </TabsContent>
+</Tabs>
+
+
+---
+
+### Override default RPC
+
+To override the default RPC for any given chain:
+
+<Tabs defaultValue="react">
+  <TabsList>
+		<TabsTrigger value="react">React</TabsTrigger>
+    <TabsTrigger value="typescript">TypeScript</TabsTrigger>
+	</TabsList>
+  <TabsContent value="react">
+    ```jsx
+    import { ThirdwebProvider } from "@thirdweb-dev/react";
+
+    const App = () => {
+      return (
+        <ThirdwebProvider
+          activeChain={{
+            rpc: ["https://<your-rpc-to-use>.com"],
+          }}
+        >
+          <YourApp />
+        </ThirdwebProvider>
+      );
+    };
+    ```
+  </TabsContent>
+	<TabsContent value="typescript">
+      ```ts
+    import { ThirdwebSDK } from "@thirdweb-dev/sdk";
+
+    const sdk = new ThirdwebSDK({
+      rpc: ["https://<your-rpc-to-use>.com"],
+    });
+    ```
+  </TabsContent>
+
+</Tabs>
+
+<Callout title="Public RPCs" variant="info">
+Public RPCs can be used without an API Key. Refer to the [configured chains page](https://thirdweb.com/dashboard/infrastructure/rpc-edge).
+</Callout>
+
+### View RPC usage
+
+[View RPC request usage](https://thirdweb.com/dashboard/settings/usage) in the dashboard settings.
+
+![Usage dashboard showing how much of each service has been used](./assets/usage-dashboard.png)
+
+<Image
+	src={require("./assets/usage-dashboard.png")}
```

Comment:
Let's change all require() to static image imports for loading image

Reply:
sg- can i address this in another PR this week